### PR TITLE
Config gen fixes - hewing closer to desired manual config

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
fixes https://github.com/googleapis/toolkit/issues/263
fixes https://github.com/googleapis/toolkit/issues/165

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
Edited description - also fixes https://github.com/googleapis/toolkit/issues/165

File: src/test/java/com/google/api/codegen/testdata/library_config.baseline [hunk 1]
```diff
@@ -282,4 +282,5 @@ interfaces:
     retry_params_name: default
     field_name_patterns:
       name: book_2
-    timeout_millis: 30000
+    timeout_millis: 60000
+
```
Comment (michaelbausor) [inline src/test/java/com/google/api/codegen/testdata/library_config.baseline]: Why is there an extra blank line here?

Comment (garrettjonesgoogle) [inline src/test/java/com/google/api/codegen/testdata/library_config.baseline]: The generator started emitting an extra blank line; I don't know why...

**Comment by michaelbausor** [comment]
LGTM

**Pull request merged**